.storybook/stories/DataFiltering/components/ActiveFilters.tsx

```diff
@@ -31,10 +31,15 @@ const getTagsFromFilters = ({
 
 	for (const [key, value] of filterEntries) {
 		if (key === 'requestDate') {
-			if (!value.from && !value.to) continue;
```

Comment:
As `value.from` and `value.from` can be undefined, string OR date, we need to only filter on this field if we have a valid date object.